routers/router.php

```diff
@@ -21,6 +21,7 @@
 
 require_once('includes/config.defaults.php');
 require_once('config.php');
+require_once 'Arista.php';
```

Comment:
Lowercase filename and add parenthesis to match code style.

Reply:
done